app/views/unattended/provisioning_templates/snippet/kickstart_kernel_options.erb

```diff
@@ -50,7 +50,7 @@ snippet: true
         options.push("ip=#{iface.ip}::#{subnet4.gateway}:#{subnet4.mask}:#{hostname}:#{iface.identifier}:none")
       end
     end
-  elsif subnet6.dhcp_boot_mode? && rhel_compatible && major >= 7
+  elsif subnet6 && subnet6.dhcp_boot_mode? && rhel_compatible && major >= 7
     if host_param_true?("use-slaac")
       options.push("ip=auto6")
     else
```

Comment:
These changes are coming from https://github.com/theforeman/foreman/pull/8145 and while I'd prefer explicit error, I think it's safer now to allow this since we have a better test coverage.